playbooks/keystone_voms.yaml

```diff
@@ -40,23 +39,24 @@
     - name: Install Keystone VOMS module
       yum: name=python-keystone_voms state=installed
 
-    - name: Add VOMS filter
-      blockinfile:
-        path: /etc/keystone/keystone-paste.ini
-        block: |
-          [filter:voms]
-          paste.filter_factory = keystone_voms.core:VomsAuthNMiddleware.factory
-      notify:
-        - restart apache
-
-    - name: Enable VOMS filter in the public_api pipeline
-      lineinfile:
-        dest: /etc/keystone/keystone-paste.ini
-        regexp: '^pipeline = (.* ec2_extension) public_service'
-        line: 'pipeline = \1 voms public_service'
-        backrefs: yes
-      notify:
-        - restart apache
+    - name: Configure VOMS filter for authentication
+      block:
+        - name: Add VOMS filter
+          blockinfile:
+            path: /etc/keystone/keystone-paste.ini
+            block: |
```

Comment:
Shouldn't there be a regexp to say where this block should go ? It seems like this may become non-idempotent.

Reply:
It's an inifile and this block is indepedant, so this shouldn't be that much a problem, but I will replace it by some inifile tasks, it will definitely be cleaner/safer.